src/table/TableWrapper.jsx

```diff
@@ -73,10 +75,44 @@ export default function TableWrapper(props) {
     };
   }, []);
 
+  useEffect(() => {
```

Comment:
If possible, would be nice to break the content of this useEffect out of the TableWrapper. Don't want to crowd this file and need to have it testable.

Reply:
Sure, I'll create a function for it to isolate the work that it does